src/app/examples/crud/components/crud.component.html

```diff
@@ -10,7 +10,7 @@ <h3 data-testid="item-title">{{book.title}}</h3>
       </mat-card>
       <p *ngIf="(books$ | async)?.length === 0">{{'anms.examples.crud.empty' | translate}}</p>
       <button type="button" mat-fab color="primary" class="add" [ngClass]="routeAnimationsElements"
-        *ngIf="!isEditing" (click)="addNew(bookForm)" data-testid="add-crud">
+        *ngIf="!isEditing" (click)="addNew(bookForm)" data-testid="add-crud" aria-label="Open a book creation form">
```

Comment:
I got this working with the binding syntax.

```html
[attr.aria-label]="'anms.examples.crud.empty' | translate"
```

Could you try this out please?